### ISSUE TITLE
Plan for monetization

### ISSUE THREAD

**Issue by Jontpan** [AxlLind/2D-platformer#99]
We should have some kind of plan for this.

- In app purchases (characters, hats)
- Ads
- Price for the app itself
- Or just completely free

What do you think?

**Comment by AxlLind** [comment]
I think the app should probably be free upfront, not many people actually pay for mobile games I don't think.

We could introduce ads after every time you die maybe? Would be nice to avoid a permanent banner showing since that takes away from the look of the game. A really bright ad at the top/bottom would look strange.

I guess we could implement in app purchases but I think that might be difficult. We could look at it in the future though.

What do you think?

**Comment by Jontpan** [comment]
Yeah that sounds good. I like having some ads though, if nothing but for the experience in how to add ads to a game. If we have ads though we should also release an adless version that costs money, or maybe remove ads as an in-app puchase.

**Comment by Jontpan** [comment]
It doesn't look too hard to enable in-app purchases, unity has a tool for it.

**Comment by Jontpan** [comment]
I'm gonna use Unity's Ad network, sounds ok? @AxlLind

**Comment by AxlLind** [comment]
Yeah that sounds good
